clustered/client/src/test/resources/configs/thin-cluster-cache-invalid-element.xml

```diff
@@ -0,0 +1,19 @@
+<eh:config
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xmlns:eh='http://www.ehcache.org/v3'
+    xmlns:tc='http://www.ehcache.org/v3/clustered'>
+
+  <eh:service>
+    <tc:cluster>
+      <tc:connection url="terracotta://example.com/myCacheManager`+"/>
+      <tc:server-side-config auto-create="false"/>
+    </tc:cluster>
+  </eh:service>
+
+  <eh:cache alias="thinCache">
+    <eh:resources>
+      <tc:clustered>xxx</tc:clustered>
```

Comment:
clearly, if we use `clustered` in XML and `thin` in code, there is a name issue.